content/about/about-content.md

```diff
@@ -2,26 +2,46 @@
 
 <!--The text at the top of this page is pulled from `eew-desc.md` in this folder-->
 
-<big>Together, we can use publicly available data to hold our government and industry accountable.</big>
 
-## Our Goals
+# About EEW
```

Comment:
Delete this line | *This title is already present because https://github.com/edgi-govdata-archiving/EEW-Website/blob/main/src/pages/about.js#L13 pulls it in from here https://github.com/edgi-govdata-archiving/EEW-Website/blob/main/content/about/about.json#L2 (it's told to look there by https://github.com/edgi-govdata-archiving/EEW-Website/blob/main/src/pages/about.js#L40)*